client/apps/edit/components/content/sections/text/index.coffee

```diff
@@ -319,7 +289,6 @@ module.exports = React.createClass
   getPlugins: ->
     plugins = []
     plugins.push({label: 'artist', style: 'artist'}) if @hasFollow
-    plugins.push({label: 'toc', style: 'toc'}) if @hasToc
     return plugins
```

Comment:
this will set off our menu to an odd number -- maybe we should add back the ordered list icon?  was removed to make space for strikethru, but functionality still exists.